Assets/MarkerMetro/Editor/MarkerMetroBuilder.cs

```diff
@@ -81,13 +81,43 @@ public static void DoBuildUniversal (string outputPath)
             {
                 throw new Exception(error); // ensures exit code != 0.
             }
+#if UNITY_EDITOR
+            //Unity creates unused assets no matter what
+            //this hack removes those assets after the compilation
+            RemoveIncorrectAssets(outputPath);
+#endif
         }
+        static void RemoveIncorrectAssets(string outputPath)
+        {
+            string projectName = PlayerSettings.productName;
+            string assetsPath = outputPath + "/" + projectName + "/" + projectName + ".Windows/Assets/";
+
+            //removing unused assets for windows
+            if (File.Exists(assetsPath + "WideTile.scale-100.png"))
+                File.Delete(assetsPath + "WideTile.scale-100.png");
+
+            if (File.Exists(assetsPath + "MediumTile.scale-100.png"))
+                File.Delete(assetsPath + "MediumTile.scale-100.png");
+
+            //removing unused assets for windowsPhone
+            assetsPath = outputPath + "/" + projectName + "/" + projectName + ".WindowsPhone/Assets/";
```

Comment:
> And this is the script that sets up the project that you are reviewing.
No, this is the script I wrote that builds the unity project, I was referring to a script that sets up the whole project, it's called `BuildScripts/Build_Local.bat`. But anyway, you are right, the output path parameter is there to do the right thing.